src/Analysis/Codelyzer.Analysis/Codelyzer.Analysis.csproj

```diff
@@ -9,13 +9,16 @@
 
     <ItemGroup>
       <PackageReference Include="CommandLineParser" Version="2.8.0" />
+      <PackageReference Include="Microsoft.Build.Utilities.Core" Version="17.1.0" />
       <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" />
-      <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="3.1.2196" />      
+      <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="3.1.2196" />
+      <PackageReference Include="MsBuildPipeLogger.Logger" Version="1.1.6" />      
```

Comment:
Do we need these? They might be leftover from the when I removed the buildalyzer nuget package and referenced the dll's directory.

Reply:
good catch.  it's just for local debugging. forgot to remove this.  thanks